src/App.jsx

```diff
@@ -1,104 +1,33 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+import { PostList } from './components/PostList/PostList';
+
+// Получение пользователей
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+// Получение комментов
+function getCommentById(postId) {
+  return (
+    commentsFromServer.filter(comment => comment.postId === postId) || null
+  );
+}
+
+// Создаём один общий объект для использования
```

Comment:
It's array, not an object
but generally, it's clear from your code (you can rename the variable to preparedPosts to make it even clearer) so I'd say this comment and 2 above are redundant

Reply:
Oh, yeah. Was writing comments for myself when I did the assignment and forgot to delete before uploading)